### PR TITLE
Different Plot speed measure

### PR DESCRIPTION
**Pull request by dgoeries**
@j9ac9k i change a bit the measure and then it becomes obvious for me between 0.11.0 and masterrr

### CODE REVIEW

File: examples/PlotSpeedTest.py [hunk 1]
```diff
@@ -5,53 +5,54 @@
...
+
     ptr += 1
-    now = time()
-    dt = now - lastTime
-    lastTime = now
-    if fps is None:
-        fps = 1.0/dt
-    else:
-        s = np.clip(dt*3., 0, 1)
-        fps = fps * (1-s) + (1.0/dt) * s
```
Comment (lidstrom83) [inline examples/PlotSpeedTest.py:44]: Any reason for taking out the moving average? Suppressing older samples allows the frame rate estimate be more responsive when changing plot options.

File: examples/PlotSpeedTest.py [hunk 2]
```diff
@@ -5,53 +5,54 @@
...
+    # Measure
+    t_start = perf_counter()
+    curve.setData(data[ptr % 10])
+    app.processEvents(QtCore.QEventLoop.AllEvents)
+
+    # Statistics
+    elapsed += perf_counter() - t_start
+    average = elapsed / ptr
+    fps = 1 / average
+    p.setTitle('%0.2f fps - %0.5fs avg' % (fps, average))
```
Comment (lidstrom83) [inline examples/PlotSpeedTest.py:46]: Elapsed time between frames would look nicer displayed in milliseconds.

File: examples/PlotSpeedTest.py [hunk 3]
```diff
@@ -5,53 +5,54 @@
 """
 
 ## Add path to library (just for examples; you do not need this)
```
Comment (lidstrom83) [inline examples/PlotSpeedTest.py:7]: This comment is referring to `import initExample` and should be kept above that line.

File: examples/PlotSpeedTest.py (hunk 1 above)
Comment (lidstrom83) [reply examples/PlotSpeedTest.py:44]: Incidentally, I find a measurable difference when replacing `s = np.clip(dt*3., 0, 1)` with `s = min(1, max(0, dt * 3.))`

Comment (dgoeries) [reply examples/PlotSpeedTest.py:44]: I will change back.

**Comment by j9ac9k** [comment]
Got super-seeded by #1610 Thanks @dgoeries for starting work on this!

**Pull request closed without merging**